packages/app/src/debug/empty/DebugBranchError.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <DebugEmptyView
+    :title="t('debugPage.emptyStates.noRunsFoundForBranch')"
+    :description="t('debugPage.emptyStates.noRunsForBranchMessage')"
+    :icon="IconTechnologyCommandLineError"
+    help-link-href="https://on.cypress.io/git-info"
```

Comment:
No UTM params needed here?

Reply:
> No UTM params needed here?

I don't see anything about UTM params in the requirements. @warrensplayer does this need added?

Reply:
Asking about this...